controllers/storageclaim_controller.go

```diff
@@ -70,6 +72,19 @@ type StorageClaimReconciler struct {
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *StorageClaimReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	ctx := context.Background()
+	if err := mgr.GetCache().IndexField(ctx, &corev1.PersistentVolume{}, pvClusterIDIndexName, func(o client.Object) []string {
+		pv := o.(*corev1.PersistentVolume)
+		if pv.Spec.CSI == nil {
+			return nil
+		}
```

Comment:
add a check for cephfs/rbd driver names as well

Reply:
done.